packages/common/client-utils/tsconfig.json

```diff
@@ -9,9 +9,5 @@
 		"rootDir": "./src",
 		"outDir": "./dist",
 		"baseUrl": ".",
-		"paths": {
-			"perf_hooks": ["types/perf_hooks.d.ts"],
-		},
-		"types": ["events"],
```

Comment:
Why isn't this needed? Is this what's causing the node types to be referenced in the api docs?

Reply:
I think the node types being referenced in the api docs is just because we're finally actually using sha.js types (which [themselves reference node types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sha.js/index.d.ts#L1)).  The api report changed not from the tsconfig change ([first commit in this PR](https://github.com/microsoft/FluidFramework/pull/18253/commits/d639c5d1f306aebcbd45be4dac6e1629e49e365b)), but when I actually started using the typed objects ([second commit in this PR](https://github.com/microsoft/FluidFramework/pull/18253/commits/03653cb2bb0a2ccdf3e5e4a6bf220d7a4f9facc7)).  I can revert the "events" change here if you like (doesn't seem to make a difference really), but it doesn't change the api docs.

Reply:
Confirmed with a messy hack that the node types do indeed source from the usage of `@types/sha.js`, specifically used to avoid explicitly typing `Hash` (guessing this was just a shortcut).

Since `Hash` doesn't actually show up on our public API surface, seems like api-extractor shouldn't strictly need to include the reference?  Though I don't know much about api-extractor internals.